phoenixRest/tests/test_app.py

```diff
@@ -58,4 +58,32 @@ def get_user(self, token):
             'Authorization': "Bearer " + token
         }), status=200)
 
-        return res.json_body
\ No newline at end of file
+        return res.json_body
+    
+    def upload_avatar(self, email, password, path, x,y, w,h, expected_status=None):
+        token, refresh = self.auth_get_tokens(email, password)
+        
+        # We get some info about the current user
+        currentUser = self.get("/user/current/", headers=dict({
+            "Authorization": "Bearer " + token
+        }), status=200).json_body
+        user_uuid = currentUser["uuid"]
+        avatar_uuid = currentUser["avatar_uuid"]
+        
+        if avatar_uuid is not None:
+            # Delete the avatar so we can test with a new one
+            # This is done so the tests can run locally
+            self.delete(f"/avatar/{avatar_uuid}/", headers=dict({
+                "Authorization": "Bearer " + token
+            }), status=200)
+        
+        upload_res = self.post(f"/user/{user_uuid}/avatar", params=f"x={x}&y={y}&w={w}&h={h}", upload_files=[("file", path)], headers=dict({
+            "Authorization": "Bearer " + token
+        }), status = (expected_status if expected_status is not None else 200))
+        
+        if expected_status is None:
```

Comment:
Yeah it definetly doesn't belong in this method as it is intended to only upload the avatar. I kept it since the avatar tests used it to delete after. Do we keep using a separate function with the old functionality in those tests or do we delete the avatars in the tests themself?

Reply:
yes keep a separate function. It's a helper that does all the test-specific logic we want for those use-cases, keeping the actual cases short and to the point